torchaudio/datasets/legacy/vctk.py

```diff
@@ -0,0 +1,243 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch.utils.data as data
+import os
+import os.path
+import shutil
+import errno
+import torch
+import torchaudio
+
+AUDIO_EXTENSIONS = [
+    '.wav', '.mp3', '.flac', '.sph', '.ogg', '.opus',
+    '.WAV', '.MP3', '.FLAC', '.SPH', '.OGG', '.OPUS',
+]
+
+
+def is_audio_file(filename):
+    return any(filename.endswith(extension) for extension in AUDIO_EXTENSIONS)
+
+
+def make_manifest(dir):
+    audios = []
+    dir = os.path.expanduser(dir)
+    for target in sorted(os.listdir(dir)):
+        d = os.path.join(dir, target)
+        if not os.path.isdir(d):
+            continue
+
+        for root, _, fnames in sorted(os.walk(d)):
+            for fname in fnames:
+                if is_audio_file(fname):
+                    path = os.path.join(root, fname)
+                    item = path
+                    audios.append(item)
+    return audios
+
+
+def read_audio(fp, downsample=True):
+    if downsample:
```

Comment:
I wonder if we can use our resample transform or if this is necessary for this dataset